src/components/content.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import mapProperties from "../properties-mapping.js";
+
+export function Content(props) {
+  return React.createElement("content", mapProperties("Content", props));
+}
+
+Content.propTypes = {
+};
```

Comment:
is this right? there are no properties?

Reply:
@SPaluchiewiczSofomo I am not sure that we have to keep Content. Sometime ago we decided to remove Content since there is no corresponding component for magic-script-components-react-native.